Source/items.cpp

```diff
@@ -4549,49 +4549,47 @@ void SortHealer()
 
 void SpawnHealer(int lvl)
 {
-	int v3;         // eax
-	ItemStruct *v4; // ebp
-	signed int v8;  // [esp-4h] [ebp-20h]
-	int v10;        // [esp+14h] [ebp-8h]
+	int i, nsi, srnd;
 
 	GetItemAttrs(0, IDI_HEAL, 1);
-	qmemcpy(healitem, item, sizeof(ItemStruct));
+	for (i = 0; i < sizeof(ItemStruct); i++) {
+		((BYTE *)&healitem[0])[i] = ((BYTE *)&item[0])[i];

```

Comment:
`healitem[0] = item[0];` didn't yeld a bin exact result in this case.

PSX does show an additional variable `ItemStruct holditem;`, but I didn't see how to apply it here.

Reply:
perhaps:
```
holditem = item[0];
healitem[0] = holditem;
```
???

Reply:
Nevermind, the `holditem` only exists on the jap build. Older PSX builds shows there was no additional item struct.

Reply:
Okay, function is bin exact when compiled as C++ is you do `healitem[0] = item[0]` ;)